railties/lib/rails/generators/test_unit/scaffold/templates/api_functional_test.rb

```diff
@@ -0,0 +1,41 @@
+require 'test_helper'
+
+<% module_namespacing do -%>
+class <%= controller_class_name %>ControllerTest < ActionController::TestCase
```

Comment:
We should replace this with an ActionDispatch::IntegrationTest generator instead. ActionController::TestCase is legacy ware.